docs/faq.rst

```diff
@@ -64,7 +64,7 @@ Does Scrapy work with HTTP proxies?
 
 Yes. Support for HTTP proxies is provided (since Scrapy 0.8) through the HTTP
 Proxy downloader middleware. See
-:class:`~scrapy.contrib.downloadermiddleware.httpproxy.HttpProxyMiddleware`.
+:class:`~scrapy.downloadermiddlewares.httpproxy.HttpProxyMiddleware`.
```

Comment:
Yeah, sounds weird, but we generally use 'middleware' to refer to a single middleware class, and 'middlewares' when we're talking of several of these classes/components in the docs. I think we're using that term in a more concrete sense, by being equal to 'middleware class'.

I'll keep the same convention for spider "middlewares" until we decide what to do with this, so we can amend the docs along with it.